recipes/postgis/meta.yaml

```diff
@@ -0,0 +1,58 @@
+{% set name = "postgis" %}
+{% set version = "2.3.3" %}
+{% set postgresql_version = "9.6.3" %}
+{% set sha256 = "723d43a1c8ca4cf06359ec0b7fd5521b355db4e37128a26fb8c661b1fe2fc930" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}+{{ postgresql_version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: http://download.osgeo.org/{{ name }}/source/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: True  # [ win ]
+
+requirements:
+  build:
+    - gdal
+    - geos
```

Comment:
You have to pin this according to https://github.com/conda-forge/conda-forge.github.io/blob/master/scripts/pin_the_slow_way.py#L64-L130